.fixtures.yml

```
@@ -6,7 +6,7 @@ fixtures:
     package: "puppetlabs/package"
     reboot: "puppetlabs/reboot"
     node_manager:
-      repo: "WhatsARanjit/node_manager"
+      repo: "puppetlabs/puppetlabs-node_manager"
```

```suggestion
      repo: "puppetlabs/node_manager"
```
shouldnt this be like that?